app/src/main/res/values/strings.xml

```diff
@@ -277,10 +277,22 @@
     <string name="title">Title</string>
     <string name="description">Description</string>
     <string name="device_info">Device info</string>
+    <string name="bug_report_use_account">Send using GitHub account</string>
     <string name="bug_report_manual">Submit the report manually</string>
     <string name="username">Username</string>
     <string name="password">Password</string>
+    <string name="bug_report_no_title">Please enter an issue title</string>
```

Comment:
maybe this should have been in the oops PR? either way it's fine

Reply:
It should, I've messed up